best-practices.md

```diff
@@ -2,20 +2,24 @@
 
 ## Table of Contents
 
-* [Enable CORS](#enable-cross-origin-resource-sharing-cors)
-* [Field and ID formatting](#field-and-id-formatting)
-* [Field selection and Metadata Linking](#field-selection-and-metadata-linking)
-* [Datetime selection](#datetime-selection)
-* [Unlocated Items](#unlocated-items)
-* [Representing Vector Layers in STAC](#representing-vector-layers-in-stac)
-* [Common Use Cases of Additional Fields for Assets](#common-use-cases-of-additional-fields-for-assets)
-* [Working with Media Types](#working-with-media-types)
-* [Static and Dynamic Catalogs](#static-and-dynamic-catalogs)
-* [Catalog Layout](#catalog-layout)
-* [Use of Links](#use-of-links)
-* [Using Relation Types](#using-relation-types)
-* [Versioning for Catalogs](#versioning-for-catalogs)
-* [STAC on the Web](#stac-on-the-web)
+* **[Web Best Practices](#web-practices)**
+  * [Enable CORS](#enable-cross-origin-resource-sharing-cors)
+  * [STAC on the Web](#stac-on-the-web)
+* **[Item Best Practices](#item-practices)**
+  * [Field and ID formatting](#field-and-id-formatting)
+  * [Field selection and Metadata Linking](#field-selection-and-metadata-linking)
+  * [Datetime selection](#datetime-selection)
+  * [Unlocated Items](#unlocated-items)
+  * [Representing Vector Layers in STAC](#representing-vector-layers-in-stac)
+* **[Asset Best Practices](#asset-practices)**
+  * [Common Use Cases of Additional Fields for Assets](#common-use-cases-of-additional-fields-for-assets)
+  * [Working with Media Types](#working-with-media-types)
+* **[Catalog Best Practices](#catalog-practices)**
+  * [Static and Dynamic Catalogs](#static-and-dynamic-catalogs)
+  * [Catalog Layout](#catalog-layout)
+  * [Use of Links](#use-of-links)
```

Comment:
Use of Links and Using Relation Types seems not so Catalog specific.

Reply:
I think it's the place they make the most sense, as it's where you do the majority of the linking. These are meant to just be rough groupings, not 'this only applies here'. I suppose we could make a 'general STAC best practices' section, but it's always going to be a bit fuzzy where to group things.

Reply:
#1006 to track this - reverted any grouping from this PR.